src/main/resources/db/changelog/petl_job_execution.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+		xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+	<changeSet id="petl_job_execution_1" author="PIH">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="petl_job_execution"/>
+		</preConditions>
+		<dropTable tableName="petl_job_execution"/>
+	</changeSet>
```

Comment:
I had originally written changesets to create the current table and then modify it to the state needed for this PR.  But that was getting unwieldy, and since we aren't really currently using this table for much of anything, I decided to purge it and recreate it.  Easier and easier to read.